src/sutta_publisher/shared/config.py

```diff
@@ -5,19 +5,107 @@
 from typing import cast
 
 import inject
+import requests
 
 from sutta_publisher.publishers.base import ActivePublishers
 from sutta_publisher.publishers.html import HtmlPublisher
 
 
+@dataclass(frozen=True)
+class EditionsConfig:
+    publication_number: str
+    text_uid: str
+    edition_number: str
+    isbn: str
+    cip: str
+    publication_type: str
+    format: str
+    creator: str
+    output: str
+    page_size: str
+    cover_image: str
+    main_toc_depth: str
+    secondary_toc: bool
+    number_of_volumes: int
+    volumes: list
+
+
 @dataclass(frozen=True)
 class Config:
-    bilara_data_url = "https://github.com/suttacentral/bilara-data"
+    PUBLICATION_JSON_RAW_URL = (
```

Comment:
This one is for internal config use only so
```suggestion
    _PUBLICATION_JSON_RAW_URL = (
```
Also, I'd put at the top of the class the most important/interesting things. The other options are more interesting, so this boring constant I'd put after the others.